tests/e2e/run_ray_trainer_rmpad.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -e -x
+
+python3 tests/e2e/arithmetic_sequence/rl/main_trainer.py \
```

Comment:
Why this CI only contains critic rmpad but not the actor?

Reply:
This is due to the misalignment between digit completion task  and the Qwen model. It may have some bug when using ChatTokenizer